docs/advanced-topics/checker-workflows.rst

```diff
@@ -28,23 +28,21 @@ information on contributing to the workflow testbed or related APIs.
 Below is a visual overview of how a checker workflow looks.
 
 .. figure:: /assets/images/docs/checker-workflow.png
-   :alt: Checker Workflow Layout
+   :alt: Flowchart describing how checker workflows work. The input is the checker workflow parameters, which go into the original tool or workflow. Next comes the validation tool or workflow. If that step exits 0, that represents concordant results, while other exit codes indicates non-concordant results. The checker workflow itself consists of both the original tool or workflow, and the validation tool or workflow.
 
-   Checker Workflow Layout
+   Checker workflow layout - note that the checker workflow "contains" the original workflow
 
 The term "entry" will be used as a generic term to refer to both tools
-and workflows.
-
-The entry that a checker workflow is testing will be referred to as an
+and workflows. The entry that a checker workflow is testing will be referred to as an
 original tool/workflow/entry.
 
-For this tutorial we will be registering a checker workflow to test an
-original tool that calculates the MD5sum of a file.
-
-The relevant tools and workflows for both CWL and WDL can be found in the following Git
+For this example, we will be registering a checker workflow to test an
+original tool that calculates the MD5sum of a file. The relevant tools and 
```

Comment:
Was like this, but I don't get what `original` means in this context.

Reply:
Probably just meant something like "the tool that we're checking" or the "original tool that we wanted to work on by adding a checker workflow"